photon-ml/src/main/scala/com/linkedin/photon/ml/algorithm/CoordinateDescent.scala

```diff
@@ -23,20 +23,21 @@ import com.linkedin.photon.ml.{BroadcastLike, RDDLike}
 import org.apache.spark.rdd.RDD
 
 /**
-  * Coordinate descent implementation
-  *
-  * @param coordinates The individual optimization problem coordinates. The coordinates is a [[Seq]] consists of
-  *                    (coordinateName, [[Coordinate]] object) pairs.
-  * @param trainingLossFunctionEvaluator Training loss function evaluator
-  * @param validatingDataAndEvaluatorOption Optional validation data evaluator. The validating data is a [[RDD]]
-  *                                         consists of (global Id, [[GameDatum]] object pairs), there the global Id
-  *                                         is a unique identifier for each [[GameDatum]] object.
-  * @param logger A logger instance
-  */
+ * Coordinate descent implementation
+ *
+ * @param coordinates The individual optimization problem coordinates. The coordinates is a [[Seq]] consists of
+ *                    (coordinateName, [[Coordinate]] object) pairs.
+ * @param trainingLossFunctionEvaluator Training loss function evaluator
+ * @param validatingDataAndEvaluatorsOption Optional validation data evaluator. The validating data is a [[RDD]]
+ *                                         consists of (global Id, [[GameDatum]] object pairs), there the global Id
+ *                                         is a unique identifier for each [[GameDatum]] object. The evaluators are
+ *                                         a [[Seq]] of evaluators
+ * @param logger A logger instance
+ */
```

Comment:
Some typos/inconsistencies here:

"... The coordinates are a [[Seq]] of (coordinateName, [[Coordinate]] object) pairs."
"... The validating data is a [[RDD]] of (globalId, [[GameDatum]] object) pairs, where globalId is a unique identifier for each [[GameDatum]] object."